src/controllers/admin/project-member-controller.ts

```diff
@@ -86,6 +87,21 @@ export const store = async (req: Request, res: Response) => {
       allocation_rate,
     })
 
+    const projectDetails = await getById(parseInt(project_id))
```

Comment:
Then please change this to `await ProjectService.getById(parseInt(project_id))`

Reply:
Okey noted :3